feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/radio.c

```diff
@@ -405,10 +405,31 @@ bool target_radio_config_set2(const struct schema_Wifi_Radio_Config *rconf,
 		blobmsg_add_u32(&b, "beacon_int", beacon_int);
 	}
 
+	LOGD("123 This function ran...");
 	if ((changed->ht_mode) || (changed->hw_mode) || (changed->freq_band)) {
 		int channel_freq;
+		char buffer[8];
+		FILE *confFile_p;
+		const char* hw_mode = rconf->hw_mode;
+
 		channel_freq = ieee80211_channel_to_frequency(rconf->channel);
-		struct mode_map *m = mode_map_get_uci(rconf->freq_band, get_max_channel_bw_channel(channel_freq, rconf->ht_mode), rconf->hw_mode);
+		// If hw_mode is "auto" determine the highest mode that we can configure this AP to be set to
+		if (!strcmp(rconf->hw_mode, "auto")) {
+			LOGD("123 auto yes");
```

Comment:
Drop this log and any other similar.

Reply:
I checked and these should be the only two.